README.md

```diff
@@ -45,7 +45,7 @@ $ mongodb-migrate --host 127.0.0.1 --port 27017 --database test --migrations exa
 You can pass on the config file that you want to use by:
 
 ```
-$ mongodb-migrate --config mig.ini
```

Comment:
If I were you, I'll try to load configurations from `config.ini` first, and then load parameters from console, and the ones from console override the ones from config file. This is what every tool normally does.

Reply:
That's what I was also thinking.. I will maybe drop in another pull request with more description in the README